source/includes/steps-realm-with-electron-using-create-react-app.yaml

```
@@ -0,0 +1,175 @@
+  point into your application. This file is responsible for loading your
+  React App's ``index.html`` file into the ``BrowserWindow`` created by
+  electron.
+
+  .. note::
+     Each electron application can only have one ``main process``. The
+     main process can create web pages, and each web page runs in it's
+     own process, known as the ``renderer process``. To learn more about
+     this read the official `Electron Application
+     Architecture <https://www.electronjs.org/docs/tutorial/application-architecture>`_
+     doc.
+
+  Add the following code to a new file called ``electron.js`` in the
+  ``public`` directory:
+
+  .. code-block:: javascript
+
+     const electron = require("electron");
+     const path = require("path");
+
+     const app = electron.app;
+     const BrowserWindow = electron.BrowserWindow;
+
+     let mainWindow;
+
+     function createWindow() {
+       // Create the browser window.
+       mainWindow = new BrowserWindow({
+         width: 800,
+         height: 600,
+         webPreferences: { nodeIntegration: true },
+       });
+       // and load the index.html of the app.
+       console.log(__dirname);
+       mainWindow.loadFile(path.join(__dirname, "../build/index.html"));
+     }
+
+     // This method will be called when Electron has finished
+     // initialization and is ready to create browser windows.
+     // Some APIs can only be used after this event occurs.
+     app.on("ready", createWindow);
+
+---
+title: Run Your Application
+ref: run-your-app
+content: |
+  In order to run your application, specify a homepage and a main entry point for electron by adding the following to your ``package.json`` file: 
+
+  .. code-block:: shell
+
+     "main": "public/electron.js",
+     "homepage": "./",
+
+  Finally, add the following scripts to your ``package.json`` file:
+
+  .. code-block:: shell
+
+     "scripts": {
+       "build": "craco build",
+       "start": "electron ."
+     },
+
+  In your terminal, run ``npm run build`` in one tab and ``npm run
+  start`` in another tab. You should see the following:
+
+  .. cssclass:: bordered-figure
+  .. figure:: /images/electron-cra.png
+    :alt: Electron Desktop App with React
+
+---
+title: Install Realm
+ref: install-realm
+content: |
+  In your terminal use the following command to add Realm to your project:
+
+  .. code-block:: shell
+
+     npm install realm@beta
+
+  You can now begin using realm in either the renderer processes or the
+  main process. 
+
+  Use realm in the renderer process by adding the following to the top
+  of the ``src/App.js`` file (you will also need to import it in whichever
+  file you write code using Realm in):
+
+  .. code-block:: shell
+
+     import Realm from "realm";
+
+  Use realm in the main process by adding the following to the top of the ``public/electron.js`` file (you will also need to require it in whichever file you write code using Realm in):
```

I would remove that parenthetical:
```suggestion
  Use Realm in the main process by adding the following to the top of the ``public/electron.js`` file:
```